schemas/external/repo/directory-page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "http://ns.adobe.com/adobecloud/core/1.0/directory-page",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Directory Page Object",
+  "type": "object",
+  "description": "Directory page object contains paging information for related directory when directory result is paginated.",
+  "definitions": {
+    "page": {
+      "properties": {
+        "orderBy": {
```

Comment:
The API spec has this all lowercase as `orderby`.  As we have naming convention to be lowerCamelCase for consistency `orderBy` would be correct.
@ogoldman, @bpruessm what do you think ?